src/_css/ModalComponent.scss

```diff
@@ -194,4 +194,6 @@
 
 .modal-container {
   height: 450px;
+  overflow-y: auto;
```

Comment:
I hope you have tested across all pages that uses this class, to ensure the change doesn't break anything

Reply:
Yes I did. Plus that class only affects the specsComponent Modal.